_data/sites.json

```
@@ -845,6 +845,17 @@
         ]
     },
 
+    {
+        "name": "Arlo",
+        "url": "https://my.arlo.com/#/settings/privacy/delete-account",
+        "difficulty": "medium",
+        "notes": "Visit the linked page and click 'Permanently Delete Arlo Account'. Click 'Continue'.  Click 'Reset and remove all devices'.  Click 'Continue'.  Click 'Delete All Content'.  Click 'Continue'.  Enter your email address and password associated with the account then click 'Permanently Delete Arlo Account'",
+        "domains": [
+            "arlo.com",
+            "my.arlo.com"
```

```suggestion
            "arlo.com"
```
Just arlo.com should be enough